engine/pkg/logutil/log_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logutil
+
+import (
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/hpcloud/tail"
+	"github.com/pingcap/log"
+	"github.com/pingcap/tiflow/engine/pkg/tenant"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+)
+
+var testFile = "test.log"
+
+func init() {
```

Comment:
In server logic, we still need to initialize global logger via the DM API, such as https://github.com/pingcap/tiflow/blob/84145a17b4ea207125c1b9410458a2b4efd428d5/engine/cmd/master/main.go#L50-L55

Is there any side effect?

Reply:
I will replace the initialize global logger part after this pr is merged.